src/constants/fuseTokenMapping.ts

```diff
@@ -46,6 +46,16 @@ export const getFuseTokenLogoURL = (address: string) => {
         uri = 'https://fuselogo.s3.eu-central-1.amazonaws.com/wfuse.png'
     }
 
+    // G$ // TODO - reads from fuse staging, handling for production?
+    if (address?.toLowerCase() === '0x100b8fd10ff8DC43fda45E636B4BB1eE6088270a'.toLowerCase()) {
+        uri = 'https://raw.githubusercontent.com/GoodDollar/GoodProtocolUI/master/src/assets/images/tokens/gd-logo.png'
+    }
+
+    // Good // TODO - reads from fuse staging, handling for production?
+    if (address?.toLowerCase() === '0xf4467Ad0c9D7D0446B379fE6217111a4690066D1'.toLowerCase()) {
+        uri = 'https://raw.githubusercontent.com/GoodDollar/GoodProtocolUI/master/src/assets/images/tokens/good-logo.png'
+    }
+
```

Comment:
on fuse G$ and GOOD have 3 different addresses for the 3 envs we have dev,staging,production
in deployment.json dev=fuse staging=staging production=production
on ropsten G$ GOOD and G$X has two addresses  fuse-mainnet and staging-mainnet
kovan G$ GOOD G$X kovan-mainnet
mainnet G$ GOOD G$X production-mainnet
@mukulshakya @adaptivecodeio

Reply:
Hi @sirpy 

Thank you for confirming the addresses. I have updated all of them please check